docs_2.0/manual_EN/3.quick-start/2.deploy-nebula-graph-with-docker-compose.md

```diff
@@ -69,27 +69,28 @@ If you are reading this topic with the questions listed below, click them to jum
 
 4. Use nebula-console to connect to Nebula Graph.
 
-    [Nebula-console](https://github.com/vesoft-inc/nebula-console) is the native CLI client of Nebula Graph. In this step, Docker pulls the nebula-console images automatically from Docker Hub according to the path you set in the following commands and uses it to connect to the graphd service of Nebula Graph. You can use other clients to connect to Nebula Graph instead of Nebula-console, such as [Nebula Graph Studio](https://github.com/vesoft-inc/nebula-web-docker) and [clients for different programming languages](https://docs.nebula-graph.io/manual-EN/1.overview/2.quick-start/3.supported-clients/).
+    [Nebula-console](https://github.com/vesoft-inc/nebula-console) is the native CLI client of Nebula Graph. In this step, Docker pulls the nebula-console images automatically from Docker Hub according to the path you set in the following commands and uses it to connect to the Graph Service of Nebula Graph.
+    <!-- You can use other clients to connect to Nebula Graph instead of Nebula-console, such as [Nebula Graph Studio](https://github.com/vesoft-inc/nebula-web-docker) and [clients for different programming languages](https://docs.nebula-graph.io/manual-EN/1.overview/2.quick-start/3.supported-clients/). -->
 
     ```bash
     $ docker run --rm -ti --network nebula-docker-compose_nebula-net vesoft/nebula-console:v2-preview-nightly --address=graphd --port=3699 -u <username> -p <password>
     ```
 
     [TODO: Update the docker image tag on the release of 2.0 alpha.]
 
-    >**NOTE**: The default username is `root` and the default password is `nebula`.
+    >**NOTE**: By default, the authentication is disabled. Use any username or password to connect Nebula Graph. The default username is `root` and the default password is `nebula`.
```

Comment:
Delete "The default username is `root` and the default password is `nebula`."?

Reply:
why? What the authentication is enabled?

Reply:
> why? What the authentication is enabled?

When the authenticati0n is enabled, use the default username `root` and its password `nebula`. ?